app/containers/message/Reply.js

```diff
@@ -146,6 +146,44 @@ const Reply = React.memo(({
 		openLink(url, theme);
 	};
 
+	if (attachment.desctiption) {
+		return (
+			<Touchable
+				onPress={onPress}
+				style={[
+					styles.button,
+					index > 0 && styles.marginTop,
+					{
+						backgroundColor: themes[theme].chatComponentBackground,
+						borderColor: themes[theme].borderColor,
+					},
+				]}
+				background={Touchable.Ripple(themes[theme].bannerBackground)}>
+				<View style={styles.attachmentContainer}>
+					<Title
+						attachment={attachment}
+						timeFormat={timeFormat}
+						theme={theme}
+					/>
+					<Description
+						attachment={attachment}
+						timeFormat={timeFormat}
+						getCustomEmoji={getCustomEmoji}
+						theme={theme}
+					/>
+					<Fields attachment={attachment} theme={theme} />
+				</View>
+				<Markdown
+					msg={attachment.description}
+					baseUrl={baseUrl}
+					username={user.username}
+					getCustomEmoji={getCustomEmoji}
+					theme={theme}
+				/>
+			</Touchable>
```

Comment:
You can just add Markdown component to Touchable Component, line 187.
`<Touchable>` should receive only one single child, so, use `<>` to wrap `attachmentContainer` and description.